thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/detection/components/MockGrouper.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.pinot.thirdeye.detection.components;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.pinot.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import org.apache.pinot.thirdeye.detection.InputDataFetcher;
+import org.apache.pinot.thirdeye.detection.annotation.Components;
+import org.apache.pinot.thirdeye.detection.annotation.DetectionTag;
+import org.apache.pinot.thirdeye.detection.annotation.Param;
+import org.apache.pinot.thirdeye.detection.annotation.PresentationOption;
+import org.apache.pinot.thirdeye.detection.spec.MockGrouperSpec;
+import org.apache.pinot.thirdeye.detection.spi.components.Grouper;
+
+
+@Components(title = "MockGrouper", type = "MOCK_GROUPER",
```

Comment:
Can you add more comment on what this MockGrouper does?

Reply:
Added